docs/tutorials/vagrant-discovery.rst

```diff
@@ -0,0 +1,376 @@
+Dynamically Discovery and Catalog Server Nodes
```

Comment:
Automatic is better, it's also aligned with http://rackhd.readthedocs.io/en/latest/rackhd/passive_discovery.html#automatic-discovery